android/src/com/mapswithme/maps/routing/NavigationController.java

```diff
@@ -281,4 +281,15 @@ public NavMenu getNavMenu()
   {
     return mNavMenu;
   }
+
+  public boolean saveState()
+  {
+    return mShowTimeLeft;
+  }
+
+  public void restoreState(boolean showTimeLeft)
+  {
+    mShowTimeLeft = showTimeLeft;
+  }
```

Comment:
Please change the signature and name of methods according my comments in the MwmActivity class.